### PR TITLE
Fix evil tests

### PR DESCRIPTION
**Pull request by nearmax**

### CODE REVIEW

File: runtime/near-vm-runner/tests/test-contract-rs/src/lib.rs [hunk 1]
```diff
@@ -290,3 +281,96 @@ pub fn sum_n() {
         value_return(data.len() as u64, data.as_ptr() as u64);
     }
 }
+
+fn build_string(n: u64) -> String {
```
Comment (evgenykuzyakov) [inline runtime/near-vm-runner/tests/test-contract-rs/src/lib.rs]: Is it just `String::from_utf8(vec![b'a'; n]);`?
@mikhailOK

Comment (evgenykuzyakov) [reply runtime/near-vm-runner/tests/test-contract-rs/src/lib.rs]: Actually you can do `"a".repeat(n)`

Comment (nearmax) [reply runtime/near-vm-runner/tests/test-contract-rs/src/lib.rs]: Fixed.

File: runtime/near-vm-runner/tests/test-contract-rs/src/lib.rs [hunk 2]
```diff
@@ -290,3 +281,85 @@ pub fn sum_n() {
...
+
+        let mut from = [0u8; size_of::<u64>()];
+        let mut to = [0u8; size_of::<u64>()];
+        from.copy_from_slice(&data[..size_of::<u64>()]);
+        to.copy_from_slice(&data[size_of::<u64>()..]);
+        let from = u64::from_le_bytes(from);
+        let to = u64::from_le_bytes(to);
+        let s = vec![b'a'; to as usize];
+        for i in from..to {
+            let mut key = s[(to - i) as usize..].to_vec();
```
Comment (evgenykuzyakov) [inline runtime/near-vm-runner/tests/test-contract-rs/src/lib.rs]: the line also can be replaced with just creating a new vec of repeated characters instead

**Pull request merged**